ansible/group_vars/do/main.yml

```
@@ -3,11 +3,14 @@ project_name: "default"
 
 main_node_size: "s-1vcpu-2gb"
 crawl_node_size: "c-4"
+admin_node_size: "s-4vcpu-8gb"
 droplet_region: "sfo3"
+enable_admin_addons: true
 
 node_pools:
   - name=main-app;size={{ main_node_size }};label=nodeType=main;count=2
   - name=crawling;size={{ crawl_node_size }};label=nodeType=crawling;taint=nodeType=crawling:NoSchedule;auto-scale=true;min-nodes=1;max-nodes=3;count=1
+  - "{% if enable_admin_addons %}name=admin-app;size={{ admin_node_size }};label=nodeType=admin;count=1{% endif %}"
```

Ah, if conditionals are allowed here, could put the whole line in conditional?
```suggestion
{% if enable_admin_addons %}
  - "name=admin-app;size={{ admin_node_size }};label=nodeType=admin;count=1"
{% endif %}
```

---

Yes, its readability isn't good. So, I tried it before. But it makes an error.

> The offending line appears to be:
> 
>   - name=admin-app;size={{ admin_node_size }};label=nodeType=admin;count=1
>         ^ here
> 
> There appears to be both 'k=v' shorthand syntax and YAML in this task. Only one syntax may be used.